packages/dx-react-scheduler-demos/src/demo-sources/scheduler-featured-appearance-customization/material-ui/demo.jsx

```diff
@@ -150,146 +148,274 @@ const styles = theme => ({
+  appointmentData, formatDate, appointmentResources,
+// #FOLD_BLOCK
+}) => {
+  const resource = appointmentResources[0];
+  const classes = useTooltipContentStyles({ color: resource.color });
+  let icon = <LowPriority className={classes.icon} />;
+  if (appointmentData.priorityId === 2) {
+    icon = <Event className={classes.icon} />;
+  }
+  if (appointmentData.priorityId === 3) {
+    icon = <PriorityHigh className={classes.icon} />;
+  }
+  return (
+    <div className={classes.content}>
+      <Grid container alignItems="flex-start" className={classes.titleContainer}>
+        <Grid item xs={2} className={classNames(classes.textCenter)}>
+          <Lens className={classNames(classes.lens, classes.colorfulContent)} />
+        </Grid>
+        <Grid item xs={10}>
+          <div>
+            <div className={classNames(classes.title, classes.dateAndTitle)}>
+              {appointmentData.title}
+            </div>
+            <div className={classNames(classes.text, classes.dateAndTitle)}>
+              {formatDate(appointmentData.startDate, { day: 'numeric', weekday: 'long' })}
             </div>
-          </Grid>
+          </div>
         </Grid>
-        <Grid container alignItems="center" className={classes.container}>
-          <Grid item xs={2} className={classes.textCenter}>
-            <AccessTime className={classNames(classes.icon, classes.grayIcon)} />
-          </Grid>
-          <Grid item xs={10}>
-            <div className={classes.text}>
-              {`${formatDate(appointmentData.startDate, { hour: 'numeric', minute: 'numeric' })}
+      </Grid>
+      <Grid container alignItems="center" className={classes.contentContainer}>
+        <Grid item xs={2} className={classes.textCenter}>
+          <AccessTime className={classes.icon} />
+        </Grid>
+        <Grid item xs={10}>
+          <div className={classes.text}>
+            {`${formatDate(appointmentData.startDate, { hour: 'numeric', minute: 'numeric' })}
               - ${formatDate(appointmentData.endDate, { hour: 'numeric', minute: 'numeric' })}`}
-            </div>
-          </Grid>
+          </div>
         </Grid>
-        <Grid container alignItems="center">
-          <Grid className={classNames(classes.contentItemIcon, priorityClasses)} item xs={2}>
-            {icon}
-          </Grid>
-          <Grid className={classes.contentItemValue} item xs={10}>
-            <span className={priorityClasses}>{` ${priority} priority`}</span>
-          </Grid>
+      </Grid>
+      <Grid container alignItems="center" key={`${resource.fieldName}_${resource.id}`}>
+        <Grid
+          className={classNames(classes.contentItemIcon, classes.icon, classes.colorfulContent)}
+          item
+          xs={2}
+        >
+          {icon}
         </Grid>
-      </div>
-    );
-  },
-);
+        <Grid item xs={10}>
+          <span className={classNames(classes.text, classes.colorfulContent)}>
+            {`${resource.text} priority`}
```

Comment:
We can omit the word 'priority' here since resource text contains this word. Now the displayed text is 'High Priority priority'